server/routes/user.ts

```diff
@@ -142,4 +143,51 @@ router.delete<{ id: string }>('/:id', async (req, res, next) => {
   }
 });
 
+router.post('/import-from-plex', async (req, res, next) => {
+  try {
+    const defaultPermission = req.body.permissions || Permission.REQUEST;
```

Comment:
We can now set default permissions through the settings page so you can grab the current `defaultPermissions` value from the settings here.